server/src/main/scala/com/cloudera/livy/server/interactive/CreateInteractiveRequest.scala

```diff
@@ -36,5 +36,6 @@ class CreateInteractiveRequest {
   var queue: Option[String] = None
   var name: Option[String] = None
   var conf: Map[String, String] = Map()
+  var scalaVersion: Option[String] = None
```

Comment:
This will only be used for user who wants to use Spark 1.6 + Scala 2.11 or Spark 2.0 + Scala 2.10. Also Spark will figure out this Scala version in future, so usually user doesn't need to take care of this field.

I could remove this field, but it will lose the ability to use Spark with unofficial Scala buildings.

Also setting Scala version as a global conf will further limit the usage. For example if we set to scala 2.10, so only Spark 1.6 and 2.0 with Scala 2.10 can be worked. In that case why do we need to ship two different repl bundles, it may not be so useful.